game-core/src/main/java/org/triplea/game/server/HeadlessServerSetup.java

```diff
@@ -127,18 +110,6 @@ ServerModel getModel() {
             });
   }
 
-  @Override
-  public void addObserver(final Observer observer) {
```

Comment:
Turns out nothing invoked 'addObserver' , without any listeners a lot of the other methods become more obviously no-ops.